exercises/exercise3.js

```diff
@@ -3,10 +3,61 @@ module.exports.ACTIVATE_BOT = false;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Coffee or Tea',
+    prompt: 'Do you like coffee or tea?',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Coffee all the way!';
     },
+    isReply: true,
+    isListening: true
   },
+  {
+    label: 'Is Ken a good teacher?',
+    prompt: 'Is #{name} a good teacher or nah?',
+    handler: function(params) {
+      return params.name + " makes me nervous when he doesn't use semicolons, but I have to remember he is a TypeScript-er"
```

Comment:
I know you already showed us a code snippet of this, but it still makes me laugh!